README.md

```diff
@@ -1,12 +1,12 @@
 # To-Do List
```

Comment:
## PR Title
![PR-Title](https://user-images.githubusercontent.com/93039359/180241494-ca95c764-b0d0-4eb5-9fe1-8a8c2a40cb17.png)

**[Optional]**
- [ ] Your PR has a title that is not so descriptive enough. Because **Testing branch** doesn't tell you what are you testing. PR title fully describes what is happening in this pull request. You could use **Testing To Do list: part 1** for your PR title. :+1: